src/main/java/org/terasology/launcher/util/DownloadUtils.java

```diff
@@ -58,6 +59,8 @@
 
     private static final Logger logger = LoggerFactory.getLogger(DownloadUtils.class);
 
+    private static final String JENKINS_URL = "http://jenkins.terasology.org";
+    private static final int JENKINS_TIMEOUT = 3000; // milliseconds
```

Comment:
3 seconds might be a bit too small, but let's keep it for now - we can still adjust this value later.